lib/DDG/Spice/Symbolab.pm

```diff
@@ -0,0 +1,23 @@
+package DDG::Spice::Symbolab;
+# ABSTRACT: Write an abstract here
+
+use DDG::Spice;
+
+spice is_cached => 1; 
+
+name "Symbolab";
+description "Get a solution to a math problem from symbolab.com";
+primary_example_queries "integral calculator", 
```

Comment:
Let's stick to queries that we can actually require us to calculate a solution. For generic queries like "integral calculator" this Instant Answer isn't any more useful than an organic result, as it only provides a link and so it shouldn't be shown.